storyblok/storyblok-design-system
# task (INT-1066): Datetime 24/12 feature + Styles updates

## Pull request type

Jira Link: [INT-1066](https://storyblok.atlassian.net/browse/INT-1066)




## How to test this PR

[Access the preview](https://storyblok-design-system-git-task-int-1066-storyblok-com.vercel.app/) and see the changes in the time component, now it is possible to see the time in AM/PM format and it is also possible to change the timezone

[Link for the design](https://www.figma.com/file/swXSVQ3pPjn6ZXeSsLDKDK/%5BSBD-154%5D-Improve-UX-date-picker-component?type=design&node-id=377-9446&mode=design&t=JZxBwaVJ3RMpUlkf-0)

https://github.com/storyblok/storyblok-design-system/assets/40925579/7a20753e-d58b-401c-b40a-f24314def72a



## What is the new behavior?
- Styles
- Possiblity to change the timezone
- AM/PM format

## Other information


[INT-1066]: https://storyblok.atlassian.net/browse/INT-1066?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
